extensions/rich_text_components/components.py

```diff
@@ -106,7 +106,7 @@ class Image(BaseRteComponent):
     def validate(cls, value_dict):
         """Validates Image component."""
         super(Image, cls).validate(value_dict)
-        filename_re = r'^[A-Za-z0-9+/]*\.((png)|(jpeg)|(gif)|(jpg))$'
+        filename_re = r'^[A-Za-z0-9+/_-]*\.((png)|(jpeg)|(gif)|(jpg))$'
```

Comment:
/cc @import-keshav this change should fix the issue you mentioned about src validation right?

Reply:
yeah right.